workflow/log-monitoring/search-filter.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Search & Filter Your Logs
+sidebarTitle: Search & Filter
+---
+
+The Search and Filter features within the Logs page provide a streamlined method for accessing your data. Create persistent filters to preserve your customized views, efficiently browse through your logs, and access important collections of logs in a single button click.
+
+To quickly filter your log data, use the leftmost column on the logs page to toggle on and off filters sorted by __Levels__, __Additional__, __NameSpace__, __APM Projects__, and __Sources__.
```

Comment:
I would change this sentence to instead describe how these filters are procured: "Middleware parses out all JSON keys from your logs as attributes, which in turn make up the filter list on the left. Any key found in your logs will appear in the left most list in alphabetical order, with the excpetion of **SeverityText** which is always at the top"